### PR TITLE
docs with Chinese

### LINKED ISSUE #296: Chinese docs

### PR DESCRIPTION
**Pull request by ye4293**
**Title:** 
docs with Chinese

**Description:** (optional)
- add README.cn.md
- add CONTRIBUTING.cn.md
- add CODE_OF_CONDUCT.cn.md
- change CODE_OF_CONDUCT.md
- A directly spliced hyperlink was used because it was not expected to be parsed into bolb/main, but instead was always parsed into tree/. If there is an easy way, you can modify it directly

**Motivation:** (optional)
- make the project more widely disseminated

**Related Issues:** (optional)
- #issue-number

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+  <strong>English</strong>|<a href="https://github.com/Portkey-AI/gateway/blob/main/CODE_OF_CONDUCT.cn.md">中文</a>
+</p>
+
 # Contributor Covenant Code of Conduct
 
 ## Our Pledge
 
 We as members, contributors, and leaders pledge to make participation in our
-community a harassment-free experience for everyone, regardless of age, body
+community a harassment-free experience for everyone, regardless of age, bodys
```
Comment (vrushankportkey) [inline .github/CODE_OF_CONDUCT.md:10]: can we retain "body"

Comment (roh26it) [inline .github/CODE_OF_CONDUCT.md:10]: @ye4293 Hi, is this a necessary change?

**Review by roh26it — CHANGES_REQUESTED** [review]
Thanks so much for the contribution! Docs in Chinese would really help with wider community!

Only 1 minor change which in my opinion wouldn't be needed in the English docs, otherwise this looks good to me.

**Review by vrushankportkey — CHANGES_REQUESTED** [review]

**Comment by vrushankportkey** [comment]
addresses #296

**Pull request merged**